client/src/pages/repo-browser/RepositoryTile.tsx

```diff
@@ -15,13 +15,27 @@ export const RepositoryTile = (props) => {
   const [isDisabled, setIsDisabled] = useState(false);
   const [isWarning, setIsWarning] = useState(false);
   const [isError, setIsError] = useState(false);
+  const [isPublic, setIsPublic] = useState(false);
```

Comment:
There is no need for this it awill not be displayed is there is no warning or errors so we can jsut skyp the validation in case there the sas token is empty

Reply:
Line 64 - 

        {!isError && !isWarning && (

I am getting the text "SAS Token Expiration: {expires}". I checked the history for the file and didn't see any changes to it, so I don't think it's an error. Adding !IsPublic removes it.